townhall/myapi/dao.py

```diff
@@ -19,3 +19,11 @@ def create_volunteer(create_volunteer_data: CreateVolunteerData):
             age=create_volunteer_data.age,
             email=create_volunteer_data.email,
         )
+
+    def delete_volunteer(volunteer_id: int) -> Volunteer:
+        try:
+            volunteer = Volunteer.objects.get(id=volunteer_id)
+            volunteer.delete()
+            return volunteer
```

Comment:
Dont need to return the volunteer object, in which case can be solidified to one line 
`Volunteer.objects.get(id=volunteer_id).delete()`